### DISCUSSION TITLE
Save default state for races other than F1

### DISCUSSION THREAD

**Discussion by drag0nst0ne** [f1multiviewer/issue-tracker#109] [Feature requests] [answered]
When opening a new stream it always defaults it to 0% volume and english audio track, By saving a layout I can change default state for all categories to my prefered settings.

**Comment by f1multiviewer** [comment]
At least the language part is now fixed :)

**Comment by jeppeolesen** [comment, marked as answer]
Language part of this request has been addressed - moving the audio part of it over to #346
